### DISCUSSION TITLE
240926 회의 : 마일스톤1 첫 전체 Merge와 이슈 공유

### DISCUSSION THREAD

**Discussion by iamheroine** [leesum-in/pagebrothers#42] [Minutes of Meetings]
## 이슈

1. svg 컴포넌트 사용
    webpack 별도 설치 후 svgr 세팅하여 컴포넌트화 해결
    
2. Prettier formatting error
    #22 
    깃허브 이슈 참조
    
3. pnpm-lock.yaml 파일에 변경점으로 conflict 발생
    - 문제 :
        push 할 때마다 conflicts 발생
    - 파악 :
        pnpm version 차이로 인한 충돌 + 이 외 다른 원인 가능성 유..
    - 해결 :
        임시 해결 방법
        ```tsx
        pnpm i --frozen-lockfile --no-verify-store-integrity
        ```
        https://github.com/pnpm/pnpm/issues/6229
        
        @leesum-in 원인 및 해결 방법 조언을 구해봅니다.
        

## 아젠다

1. 공통 컴포넌트 className props
    필요 시 추가하기
    
2. 컴포넌트 import 상대 경로 사용
    최상위 레벨 index.tsx에도 추가해서 사용하기
![image](https://github.com/user-attachments/assets/0029fc3e-acd7-459b-8eb4-d19968e5a397)
    
3. 함수 표현식 vs 함수 선언식
    discussion에 올려서 빠른 시일 내로 정하기
    #41 
    
4. 코드 컨벤션 추가
    - 컴포넌트 import 상대 경로 사용
        최상위 레벨 index.tsx에도 추가해서 사용하기
        ![image](https://github.com/user-attachments/assets/0029fc3e-acd7-459b-8eb4-d19968e5a397)
    - react import 사용하지 않기
    - React.FC 사용하지 않기
        [[React.FC를 사용하지 않는 이유](https://yceffort.kr/2022/03/dont-use-react-fc)]
        
    @YISYISYISYIS 컨벤션에 업데이트 부탁드립니다. (FC는 사용하지 않기로 결정한 이유도 간략하게 추가해주시면 감사하겠습니다!)

## 이 외 기타

1. 동규님 하차 후 태스크 분배, 일정 조율
    2024년 9월 28일 오후 7:00 회의
    
2. 이번주 토요일까지 마일스톤 1 완료를 목표

**Comment by YISYISYISYIS** [comment]
@YISYISYISYIS 컨벤션에 업데이트 부탁드립니다. (FC는 사용하지 않기로 결정한 이유도 간략하게 추가해주시면 감사하겠습니다!)

...? 이거 제가하는거에요?
아 내가 쓴글이라 수정은 나만 하는건가😂

**Reply by iamheroine** [reply]
엇 인수님이 하시는걸로 알고 있었는데 아닌가요? 앞으로 추가될 일이 또 있을텐데 담당을 정하면 좋겠네요!

**Reply by YISYISYISYIS** [reply]
도희님이 까라고 하면까야죠ㅠ 
컨벤션 수정했습니다😭
